functional_test.go

```diff
@@ -1768,3 +1837,46 @@ func waitForBroadcast(timeout clock.Duration, d *guber.Daemon, expect int) error
 		}
 	}
 }
+
+// staticBuilder implements the `resolver.Builder` interface.
+type staticBuilder struct{}
+
+func newStaticBuilder() resolver.Builder {
```

Comment:
`newStaticBuilder` already exists in `deamon.go` perhaps we should make it public and use it in the above test?